CategoriesPanel.c

```
@@ -36,6 +36,8 @@ static const char* const CategoriesFunctions[] = {"      ", "      ", "      ",
 
 static void CategoriesPanel_delete(Object* object) {
    CategoriesPanel* this = (CategoriesPanel*) object;
+   if (this->header->metersCopied)
```

Cppcheck (2.20.0)

[style] The function 'Vector_quickSort' is never used.

(unusedFunction)